src/views/Association.vue

```diff
@@ -8,7 +8,7 @@ import Content from '@/components/Content.vue';
     L'association InsaLan
   </h1>
   <Content
-    class="flex flex-col gap-8 indent-12 lg:px-24"
+    class="indent-content flex flex-col gap-8 lg:px-24"
```

Comment:
Is there a reason to use css classe instead of tailwind ?
You can use tailwind target selection otherwise : https://stackoverflow.com/questions/67119992/how-to-access-all-the-direct-children-of-a-div-in-tailwindcss

Reply:
I didn't know we can do this. 😅